python/py-terminaltables/Portfile

```diff
@@ -15,7 +15,8 @@ description         Generate simple tables in terminals from a nested list of st
 long_description    Easily draw tables in terminal/console applications from \
                     a list of lists of strings. Supports multi-line rows.
 
-python.versions     37 38 39 310
+python.rootname     terminaltables
```

Comment:
this is the same as is set by the `python` PortGroup based on the `name` so please remove

Reply:
Doing that now, thanks!